editor/src/clj/editor/sprite.clj

```diff
@@ -592,13 +592,24 @@
 
   (input copied-nodes g/Any :array :cascade-delete)
 
-  (output aabb AABB (g/fnk [size]
+  (output aabb AABB (g/fnk [size ^:try scene-infos]
                       (let [[^double width ^double height ^double depth] size
+                            first-animation (when-not (g/error-value? scene-infos)
+                                              (:animation (first scene-infos)))
+                            anim-pivot (when first-animation
+                                         (get-in first-animation [:frames 0 :pivot]))
+                            [pivot-x pivot-y] (if anim-pivot
+                                                anim-pivot
+                                                [0.0 0.0])
                             half-width (* 0.5 width)
                             half-height (* 0.5 height)
-                            half-depth (* 0.5 depth)]
-                        (geom/make-aabb (Point3d. (- half-width) (- half-height) (- half-depth))
-                                        (Point3d. half-width half-height half-depth)))))
+                            half-depth (* 0.5 depth)
+
+                            ; Adjusted calculations for min-x and min-y
```

Comment:
Pivot should be taken into account when calculate AABB otherwise when you open scene with one sprite and custom pivot point, you don't see sprite on the scene